### PR TITLE
Support Cross-origin resource sharing.

### PR DESCRIPTION
**Pull request by slashrsm**

### CODE REVIEW

**Comment by slashrsm** [comment]
Still needs tests.

File: src/app.js [hunk 1]
```diff
@@ -30,60 +30,128 @@ app.post('/', async (req: $Subtype<express$Request>, res: express$Response) => {
...
-    res.status(201).json({})
+    res
+      .status(201)
+      .json({})
+      .set('Access-Control-Allow-Origin', '*')
+      .set('Access-Control-Allow-Methods', 'POST')
   } catch (err) {
-    res.status(400).json({ error: err.message })
+    res
+      .status(500)
```
Comment (slashrsm) [inline src/app.js]: I feel that "Internal server error" is more appropriate in this case?

File: src/app.js [hunk 2]
```diff
@@ -30,60 +30,128 @@ app.post('/', async (req: $Subtype<express$Request>, res: express$Response) => {
...
 app.get(
   '/un-p2sh/:txId',
   async (req: $Subtype<express$Request>, res: express$Response) => {
     try {
       const sql = 'SELECT output_data FROM UnP2sh WHERE tx_id = ${tx_id}'
       const result: Array<any> = await Db.any(sql, objToSnakeCase(req.params))
       const data = JSON.parse(result[0].output_data)
-      res.status(201).json(data.map(objToCamelCase))
+      res
+        .status(200)
```
Comment (slashrsm) [inline src/app.js]: 201 is "Created" so I think 200 "OK" fits GET requests better.

File: src/app.js [hunk 3]
```diff
@@ -30,60 +30,128 @@ app.post('/', async (req: $Subtype<express$Request>, res: express$Response) => {
...
+        .json(data.map(objToCamelCase))
+        .set('Access-Control-Allow-Origin', '*')
+        .set('Access-Control-Allow-Methods', 'GET')
     } catch (err) {
-      res.status(400).json({ error: err.message })
+      res
+        .status(404)
```
Comment (slashrsm) [inline src/app.js]: 404 "Not found" seems better than 400 "Bad request" for GET requests

**Comment by slashrsm** [comment]
I tested this in the browser and fixed few things. Can confirm that this version gets rid of the CORS error. Also added test coverage. 

There is also https://www.npmjs.com/package/cors, which would make the code much simpler. We could also use that, but I am not sure if we want to  introduce another dependency.

**Comment by ClemensLey** [comment]
Looks good

**Pull request merged**